app/views/portal/show.rb

```diff
@@ -422,21 +414,19 @@ def labels
 
     private
 
-    def use_edm_is_shown_by_proxy?
+    def use_media_proxy?(web_resource_url, mime_type)
```

Comment:
Unused method argument - `web_resource_url`. If it's necessary, use `_` or `_web_resource_url` as an argument name to indicate that it won't be used.